packages/expo-video/android/src/main/java/expo/modules/video/player/VideoPlayerAudioTracks.kt

```diff
@@ -0,0 +1,125 @@
+  private val owner = WeakReference(owner)
+  private val videoPlayer: VideoPlayer?
+    get() {
+      return owner.get()
+    }
+  private val formatsToGroups = mutableMapOf<Format, Pair<TrackGroup, Int>>()
+  private var currentAudioTrackFormat: Format? = null
+  private var currentOverride: TrackSelectionOverride? = null
+
+  var currentAudioTrack: AudioTrack?
+    get() {
+      return AudioTrack.fromFormat(currentAudioTrackFormat)
+    }
+    set(value) {
+      applyAudioTrack(value)
+    }
+  val availableAudioTracks = arrayListOf<AudioTrack>()
+
+  init {
+    owner.addListener(this)
+  }
+
+  fun setAudioTracksEnabled(enabled: Boolean) {
+    val currentParams = videoPlayer?.player?.trackSelectionParameters ?: return
+    var params = currentParams.buildUpon().setTrackTypeDisabled(C.TRACK_TYPE_AUDIO, !enabled).build()
+    if (!enabled) {
+      params = params.buildUpon().clearOverridesOfType(C.TRACK_TYPE_AUDIO).build()
+    }
+    videoPlayer?.player?.trackSelectionParameters = params
+  }
+
+  // VideoPlayerListener
+  override fun onTrackSelectionParametersChanged(player: VideoPlayer, trackSelectionParameters: TrackSelectionParameters) {
+    currentAudioTrackFormat = findSelectedAudioFormat()
+    super.onTrackSelectionParametersChanged(player, trackSelectionParameters)
+  }
+
+  override fun onTracksChanged(player: VideoPlayer, tracks: Tracks) {
+    formatsToGroups.clear()
+    availableAudioTracks.clear()
+    for (group in tracks.groups) {
+      for (i in 0..<group.length) {
+        val format: Format = group.getTrackFormat(i)
+
+        if (MimeTypes.isAudio(format.sampleMimeType)) {
+          formatsToGroups[format] = Pair(group.mediaTrackGroup, i)
+          val track = AudioTrack.fromFormat(format) ?: continue
+          availableAudioTracks.add(track)
+        }
+      }
+    }
+    currentAudioTrackFormat = findSelectedAudioFormat()
+    super.onTracksChanged(player, tracks)
+  }
+
+  // Private methods
+  private fun applyAudioTrack(audioTrack: AudioTrack?) {
+    val player = videoPlayer?.player ?: return
+    var newParameters: TrackSelectionParameters = player.trackSelectionParameters
+    currentOverride?.let { override ->
+      newParameters = newParameters.buildUpon().clearOverridesOfType(C.TRACK_TYPE_AUDIO).build()
+    }
+    if (audioTrack == null) {
+      player.trackSelectionParameters = newParameters
+      setAudioTracksEnabled(false)
+      currentOverride = null
+      return
+    }
+    val format = formatsToGroups.keys.firstOrNull {
+      it.id == audioTrack.id
+    }
+    format?.let {
+      formatsToGroups[it]?.let { subtitlePair ->
+        val override = TrackSelectionOverride(subtitlePair.first, subtitlePair.second)
+        newParameters = newParameters.buildUpon().addOverride(override).build()
+        player.trackSelectionParameters = newParameters
+        setAudioTracksEnabled(true)
+        currentOverride = override
```

Comment:
```suggestion
        val trackSelectionOverride = TrackSelectionOverride(subtitlePair.first, subtitlePair.second)
        newParameters = newParameters.buildUpon().addOverride(trackSelectionOverride).build()
        player.trackSelectionParameters = newParameters
        setAudioTracksEnabled(true)
        currentOverride = trackSelectionOverride
```
I wouldn't use override as it's a key word in Kotlin.